core/components/atoms/page/__stories__/index.story.jsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Page, Heading, Text } from '@/index';
+
+// CSF format story
+export const all = () => {
+  return (
+    <Page aria-label="Sample Page" padding="large">
```

Comment:
In Page component, prop `padding` type is defined as string, then how would user know that we need to pass "large" here?